CHANGELOG.md

```diff
@@ -442,6 +442,13 @@ Other minor additions
   splitAt-≥ : ∀ m {n} i → (i≥m : toℕ i ℕ.≥ m) → splitAt m {n} i ≡ inj₂ (reduce≥ i i≥m)
   ```
 
+* Added new proofs to `Data.Vec.Properties`:
+  ```agda
+  take-step : ∀ n {m} x (xs : Vec A (n + m)) → take (suc n) (x ∷ xs) ≡ x ∷ take n xs
```

Comment:
Looking at the other PR, I was reminded that the similar-looking
unfolding lemma for `reverse` is called `unfold-reverse` so this
one should probably be called `unfold-take`.